.ado/windows-vs-pr.yml

```diff
@@ -601,7 +601,7 @@ jobs:
     cancelTimeoutInMinutes: 5
     steps:
       - checkout: self
-        fetchDepth: 1
+        fetchDepth: 15
```

Comment:
>15 [](start = 20, length = 2)

I wonder about the arbitrary number 15. For the CI machines it should be no different than number 1 or 2.